Lib/pdb.py

```diff
@@ -96,7 +97,14 @@ class Restart(Exception):
     pass
 
 __all__ = ["run", "pm", "Pdb", "runeval", "runctx", "runcall", "set_trace",
-           "post_mortem", "help"]
+           "post_mortem", "help", "PdbInvokeType"]
+
+
+class PdbInvokeType(enum.Enum):
```

Comment:
Don't use type, it's overloaded.   How about InvocationMethod?

Reply:
I hesitated about type as well, but method is also heavily overloaded right? It's a function of an instance. Also the name should not be too long I guess? I bet there are better names out there.